content/blog/cemiterio-de-elefantes/index.md

```
@@ -0,0 +1,193 @@
+
+![A espécime conhecida como desenvolvedor - Fonte Instagram.](um-desenvolvedor.png)
+
+## A Carreira em Y: Escolhas para Profissionais Sêniores
+
+O conceito de carreira em Y possibilita que profissionais sêniores escolham
+entre duas trajetórias principais: a gestão de pessoas e processos ou a
+excelência técnica. Essa segunda opção tem ganhado mais estrutura e
+reconhecimento nas últimas décadas, especialmente em áreas como a engenharia de
+software. Vamos explorar algumas dessas possibilidades:
+
+#### 1. **Contribuidor Individual (Individual Contributor)**
```

markdownlint-cli2 (0.17.2)

Heading levels should only increment by one level at a time
Expected: h3; Actual: h4

(MD001, heading-increment)